### ISSUE TITLE
spending advice

### ISSUE THREAD

**Issue by melindaloubser1** [RasaHQ/financial-demo#34]
Add skill for spending advice e.g. 
"How can I manage my money better?"
"What can I do to save more?"

https://github.com/RasaHQ/customers/issues/185

**Comment by hsm207** [comment]
@melindaloubser1 this looks interesting and I'd like to work on it if it is still open. 

The link in the description leads to 404. Is this linked to an internal repo?

**Comment by melindaloubser1** [comment]
Yes, it's an internal repo link, but if you'd like to contribute that's great! The idea is to add a simple saving advice skill, probably independent of a user's profile or account. If you'd like to add this, I'd suggest starting with the  `rasa-2-0` branch since master is still on 1.10.x.

**Comment by andrelucasf** [comment]
I found it interesting and would like to contribute to this question.
Is it possible to provide more information?

**Comment by melindaloubser1** [comment]
@andrelucasf There isn't one right way to do this, as it's a general enhancement suggestion for a new skill. I'd suggest starting with the idea of "A user asks for spending advice" and working through what would be needed from there. 
E.g. you'll need NLU training data for how users might ask this question, assigned to an intent, and create an action or response template for what advice to give them.
You could start with a really general case, where the bot always returns static advice, then if you wanted you could take it further to using a custom action that considers the users spending history (for example) and then decides what advice to give. 
If you have a different idea for how to do it feel free to suggest!
